docs/docsite/rst/scenario_guides/vmware_scenarios/vmware_intro.rst

```diff
@@ -21,15 +21,15 @@ and vCenter infrastructure. You can install pyVmomi using pip:
 
 .. code-block:: bash
 
-    $ pip install pyvmomi
+    $ pip3 install pyvmomi
```

Comment:
Thanks @jakelevinez I'd prefer to leave the code example using `pip` but update the line above to say:
"You can install pyVmomi using pip (you may need to use `pip3`, depending on your OS/distro):